main.go

```diff
@@ -0,0 +1,272 @@
+package main
+
+import (
+	"context"
+	"fmt"
+	"log/slog"
+	"math"
+	"math/big"
+	"math/rand"
+	"os"
+	"time"
+
+	"github.com/ethereum/go-ethereum/core/types"
+	"github.com/ethereum/go-ethereum/ethclient"
+	ee "github.com/primev/preconf_blob_bidder/internal/eth"
+	bb "github.com/primev/preconf_blob_bidder/internal/mevcommit"
+	"github.com/urfave/cli/v2"
+)
+
+// Define flag names as constants
+const (
```

Comment:
Instead of this, I'd suggest doing the following:
```Golang
var (
    bidderAddressFlag := &cli.StringFlag{
        Name:    "bidder-address",
        Usage:   "Address of the bidder",
        EnvVars: []string{"BIDDER_ADDRESS"},
        Value:   "mev-commit-bidder:13524",
     },
     // other flags...
)

// Now below you can do:
c.String(bidderAddressFlag.Name)